portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -80,16 +90,20 @@ public void createEntitys(){
     datastore.put(commentEntity);
   }
 
-  public void populateList(PreparedQuery results){
+  public void displayMaxComments(PreparedQuery results, int maxComment){
     this.comments = new ArrayList<>();
-    for (Entity commentEntity : results.asIterable()) {
+    for (Entity commentEntity : results.asIterable()){
+      if(maxComment == 0){
+          break;
+      }
       Comment comment = new Comment(
         commentEntity.getKey().getId(), 
         (String) commentEntity.getProperty(USERNAME_PARAMETER), 
         (String) commentEntity.getProperty(REFLECTION_PARAMETER),
         (long) commentEntity.getProperty(TIMESTAMP_PARAMETER)
         );
       comments.add(comment);
+      maxComment--;
```

Comment:
I wouldn't modify member variables here. It is an unnecessary side-effect.

I think this could be broken down easier:
```
for (Entity commentEntity : Iterables.limit(results.asIterable(), maxComment)) {
    comments.add(createCommentFromEntity(commentEntity));
}
```

Reply:
The `Iterables` is from the Google Gauva package, which I think can be installed via maven.

See: https://guava.dev/releases/21.0/api/docs/com/google/common/collect/Iterables.html